articles/tutorials/circuit-tutorial.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Tutorial: Simulate a quantum circuit | Microsoft Docs
```

Comment:
Q# aims to be more than a quantum circuits language, as such, it would be better if instead of calling this "quantum circuits" we call them "quantum programs" or "quantum algorithms"

Reply:
Absolutely, thanks! Don't hold back when it comes to those terminology/phrasing clarifications, because I'd much rather be fixing the debt than contributing to it (e.g. just yesterday Chris mentioned that we're going with classical "host" over "driver", so that'll be changed herein as well).